graphql_api/types/plan_representation/plan_representation.py

```diff
@@ -14,27 +14,27 @@
 
 @plan_representation_bindable.field("marketingName")
 def resolve_marketing_name(plan_data: PlanData, info) -> str:
```

Comment:
Did we update the PlanData type somewhere to reflect this? Can't remember if that is included in shared

Reply:
oh yes! shared changes included this, we added convet_to_DTO fn to the type

Reply:
Right but that's just a return type of that function right? The PlanData type still reflects the explicit properties of the dataclass iirc

Reply:
I don't remember if we actually created an interface for convert_to_DTO

Reply:
hmmm yah it's a bit confusing because we're kinda calculating it on the fly within the class and actually returning a dict type after all, we could enhance this at some point to make the typing clearer, but it's not gonna effect things atm